### PR TITLE
[KYUUBI #4376] Support to config the kyuubi service administrator with kyuubi conf

### LINKED ISSUE #4376: [FEATURE] Support to config the kyuubi service administrator with kyuubi conf
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the feature

Support to config the kyuubi service administrator with kyuubi conf.

Now there is only one administrator.

```
private[v1] class AdminResource extends ApiRequestContext with Logging {
  private lazy val administrator = Utils.currentUser
```

We can add a config for this, so that we can grant permission to other service accounts.

### Motivation

_No response_

### Describe the solution

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [ ] Yes. I would be willing to submit a PR with guidance from the Kyuubi community to improve.
- [ ] No. I cannot submit a PR at this time.

### PR DESCRIPTION
**Pull request by lightning-L**
### _Why are the changes needed?_

Close #4376

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by lightning-L** [comment]
@turboFei

File: docs/deployment/settings.md [hunk 1]
```diff
@@ -451,6 +451,7 @@ You can configure the Kyuubi properties in `$KYUUBI_HOME/conf/kyuubi-defaults.co
 
 |                           Key                            |      Default      |                                                                                                                    Meaning                                                                                                                     |   Type   | Since |
 |----------------------------------------------------------|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|-------|
+| kyuubi.server.admin.users                                                   || Comma-separated list of Kyuubi service admin users. We use this config to grant admin permission to any service accounts.                                                                                                                      | seq      | 1.8.0 |
```
Comment (turboFei) [inline docs/deployment/settings.md:454]: how about just `kyuubi.server.administrators`?

A comma-separated list of Kyuubi service administrators.

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala [hunk 2]
```diff
@@ -54,7 +55,7 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
     val userName = fe.getSessionUser(Map.empty[String, String])
     val ipAddress = fe.getIpAddress
     info(s"Receive refresh Kyuubi server hadoop conf request from $userName/$ipAddress")
-    if (!userName.equals(administrator)) {
+    if (!administrators.contains(userName)) {
```
Comment (turboFei) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala:58]: we can add a method for this. 

```
isAdministrator
```

File: kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala [hunk 3]
```diff
@@ -2429,6 +2429,16 @@ object KyuubiConf {
...
 
+  val SERVER_ADMIN_USERS: ConfigEntry[Seq[String]] =
+    buildConf("kyuubi.server.admin.users")
+      .doc("Comma-separated list of Kyuubi service admin users. " +
+        "We use this config to grant admin permission to any service accounts.")
+      .version("1.8.0")
+      .serverOnly
+      .stringConf
+      .toSequence()
+      .createWithDefault(Seq.empty)
```
Comment (turboFei) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala:2440]: nit: createWithDefault(Nil)

**Review by turboFei — APPROVED** [review]
LGTM

**Comment by turboFei** [comment]
thanks, merged to master

**Pull request closed without merging**